great_expectations/self_check/util.py

```diff
@@ -1499,20 +1537,14 @@ def build_test_backends_list(
             test_backends += ["mssql"]
 
         if include_bigquery:
-            gcp_project = "superconductive-internal"
-            bigquery_dataset = "demo"
-            table = "taxi_data" # not used in this context
-            # how do you get rid of credentials_path
             try:
-                engine = create_engine(f"bigquery://{gcp_project}/{bigquery_dataset}", credentials_path='/Users/work/Development/creds/superconductive-internal-ba8ee4857de2.json'
-
-                )
+                engine = _create_bigquery_engine()
                 conn = engine.connect()
                 conn.close()
-            except (ImportError, sa.exc.SQLAlchemyError):
+            except (ImportError, sa.exc.SQLAlchemyError) as e:
                 raise ImportError(
-                    "bigquery tests are requested, but unable to connect "
-                )
+                    "bigquery tests are requested, but unable to connect"
+                ) from e
```

Comment:
Including the cause with `from` is very helpful for debugging and it would be helpful if GE did that whenever catching and re-raising an exception!

Reply:
:) thank you very much. I'll start making some noise about this